proposed/2022/dotnet-nuget-why-proposal.md

```diff
@@ -0,0 +1,139 @@
+# dotnet nuget why command
+
+- Author: [Kartheek Penagamuri](https://github.com/kartheekp-ms), [Pragnya Pandrate](https://github.com/pragnya17)
+- GitHub Issue [11782](https://github.com/NuGet/Home/issues/11782)
+
+## Summary
+
+Currently, there is not a great solution for developers to understand the nature of top-level packages and their transitive dependencies. The solution explorer in Visual Studio does a decent job at allowing a user to dive into each NuGet package & see all transitive dependencies from the top-level dependency.
+
+![](../../meta/resources/TransitiveDependencies/SolutionView.png)
+
+However, in comparison to Visual Studio, the dotnet CLI does not provide insight into “why” a transitive dependency is listed, although it does list everything that has been resolved as seen below.
+
+![](../../meta/resources/TransitiveDependencies/DotNetCLI.png)
+
+## Motivation
+
+A developer should be able to understand where every package in a solution/project originated from. This can help them understand complex dependency chains they may have within their projects. Having this understanding can help a developer promote a transitive dependency quickly & easily to a top-level dependency in case there is a security concern or conflict.
```

Comment:
Great motivation here.

Reply:
Thanks to @kartheekp-ms for coming up with it!